packages/web-app-files/tests/unit/components/Spaces/SpaceContextActions.spec.ts

```diff
@@ -36,7 +37,8 @@ function getWrapper(space) {
       global: {
         mocks: {
           ...defaultComponentMocks({ currentRoute: mock<RouteLocation>({ path: '/files' }) }),
-          $can: () => true
+          $can: () => true,
+          $thumbnailService: mock<ThumbnailService>({ getSupportedMimeTypes: jest.fn(() => []) })
```

Comment:
Add to defaultComponentMocks?

Reply:
And this likely conflicts with the space action composable PR, can we merge that one first? 👀

Reply:
... it actually might not conflict if you just add it to defaultComponentMocks 😁